common/artifacts/000_wc_artifacts.txt

```diff
@@ -714,10 +1003,11 @@ book_of_medivh = {
 	stacking = no
 	indestructible = yes
 
+	diplomacy = 1
 	learning = 7
```

Comment:
Ooof, it gives so many attributes. I wouldn't add more.

Reply:
ok, done